### ISSUE TITLE
Camunda version 7.7.0

### ISSUE THREAD

**Issue by httpdigest** [camunda/camunda-bpm-spring-boot-starter#254]
Will there be a Spring Boot Starter for Camunda Engine version 7.7.0?

Also: With Spring Boot starters it is always possible to override the versions of required dependencies, such as Hibernate or Tomcat via Maven Properties "hibernate.version" or "tomcat.version", respectively.

The Camunda Spring Boot starter has `camunda-bpm-spring-boot-starter-root` as Maven ancestor (parent of its parent) and this artifact defines the `camunda.version` property (currently with default value 7.6.0). However it does not seem possible for an application's pom.xml to override this `camunda.version` to anything else.
There will also be 7.6.0 used.

It'd be nice if the Camunda Spring Boot Starter allows for the same dependency version override configuration as do all other Spring Boot Starters.

**Comment by jangalinski** [comment]
the next release 2.2 will use camunda 7.7.

Overriding the version the way you described should be possible, that's how you switch to *-ee for example ... I will have to investigate this, if it's not possible, it's a bug

**Comment by jangalinski** [comment]
Hello Kai. Just upgraded to 7.7, seams like no changes to the code base are required. So right now, the spring boot starter 2.x supports 7.6 and 7.7 version.

For your other point "changing the 'default' camunda version" I created PoC that still works with 7.6.0.

In the documentation there is a section in "getting started" that describes how to switch the version, Unfortunately, it is not enough to set the camunda.-version property, you also have to provide a dependencyManagement reference to the camunda-bom in the required version.

This works. I do not like it myself, I would prefer just switching the property, but I am not sure if we are able to support this with our current setup. That's why I created an issue to investigate this, see #257.
